.ci/Jenkinsfile

```diff
@@ -161,6 +160,7 @@ pipeline {
       when {
         beforeAgent true
         allOf {
+          not { tag pattern: 'v\\d+\\.\\d+\\.\\d+', comparator: 'REGEXP' }
```

Comment:
Down in L376 this was slight more permissive with a wildcard match on the end. I suspect this format will be fine but just wanted to confirm with you that you agree.

Reply:
Noted. We haven't used any pre-release version tags since Alan and I started maintaining this repo. I'm fine with the simpler `v\d+\.\d+\.\d+` for now. Thanks.